bin/node/cli/src/chain_spec.rs

```diff
@@ -342,10 +342,11 @@ pub fn testnet_genesis(
 		babe: BabeConfig {
 			authorities: vec![],
 			epoch_config: Some(kitchensink_runtime::BABE_GENESIS_EPOCH_CONFIG),
+			..Default::default()
 		},
 		im_online: ImOnlineConfig { keys: vec![] },
-		authority_discovery: AuthorityDiscoveryConfig { keys: vec![] },
-		grandpa: GrandpaConfig { authorities: vec![] },
+		authority_discovery: AuthorityDiscoveryConfig { keys: vec![], ..Default::default() },
+		grandpa: GrandpaConfig { authorities: vec![], ..Default::default() },
```

Comment:
```suggestion
		grandpa: GrandpaConfig::default(),
```
Same above for auth discovery. And also the babe `authorities` field can be removed (vec![] == default_

Reply:
done